crates/bevy_render/src/camera/camera.rs

```diff
@@ -32,9 +31,8 @@ use wgpu::Extent3d;
 /// The viewport defines the area on the render target to which the camera renders its image.
 /// You can overlay multiple cameras in a single window using viewports to create effects like
 /// split screen, minimaps, and character viewers.
-// TODO: remove reflect_value when possible
-#[derive(Reflect, FromReflect, Debug, Clone, Serialize, Deserialize)]
-#[reflect_value(Default, Serialize, Deserialize)]
+#[derive(Reflect, FromReflect, Debug, Clone)]
+#[reflect(Default)]
```

Comment:
I think a quick migration guide (or at least a more prominent note) in the PR description to indicate this breaking change would be good. Not only is it not `Serialize`/`Deserialize` anymore, but it's also not `reflect_value`, which means that it needs to be treated as a `ReflectRef::Struct` as well.